History.md

```diff
@@ -1,6 +1,8 @@
 unreleased
 ==========
 
+  * deps: cookie@0.4.1
+        - Fix `maxAge` option to reject invalid values.
```

Comment:
This sub list is indented a bit too much (see the other version entries for example).

Reply:
typo :/